microtrac_gazebo/CMakeLists.txt

```diff
@@ -1,6 +1,15 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(microtrac_gazebo)
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS roslaunch)
+
 catkin_package()
+catkin_add_env_hooks(50.microtrac_gazebo
+  SHELLS sh DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/env-hooks)
```

Comment:
Shell scripts in cmake files are a bit funky. Is this to construct the URDF?

Reply:
This came from Husky. I removed it in my latest commit in favor of a find in the launch file.